src/platform/utilities/date/index.js

```diff
@@ -78,3 +78,49 @@ export function timeFromNow(date, userFromDate = null) {
 export function isValidDateString(dateString) {
   return !isNaN(Date.parse(dateString));
 }
+
+const monthIndices = {
+  JAN: 0,
+  FEB: 1,
+  MAR: 2,
+  APR: 3,
+  MAY: 4,
+  JUN: 5,
+  JUL: 6,
+  AUG: 7,
+  SEP: 8,
+  OCT: 9,
+  NOV: 10,
+  DEC: 11,
+};
+
+const LONG_FORM_MONTHS = [
+  monthIndices.MAR,
+  monthIndices.APR,
+  monthIndices.MAY,
+  monthIndices.JUN,
+  monthIndices.JUL,
+];
+
+export const formatDowntime = dateTime => {
+  const dtMoment = moment.parseZone(dateTime);
```

Comment:
Ooh, didn't realize [`parseZone`](https://momentjs.com/docs/#/parsing/parse-zone/) was a thing for plain ol' `moment`. Good call. :+1:

Reply:
Me neither until I realized the specs were broken. 😄